wheatley/main.py

```diff
@@ -326,9 +326,9 @@ def console_main(override_args: Optional[List[str]], stop_on_join_tower: bool) -
         default="1234",
         help='An override for what place notation(s) should be made when a `Single` is called in \
               Ringing Room.  These will by default happen at the lead end.  Examples: "1678" or \
-              "0:1678" => 6ths place lead end single.  "-1:3.123" => a Grandsire Single. \
-              "20: 7890" => a 7890 single taking effect 20 changes into a lead (the Half Lead for \
-              Surprise Royal). "3: 567/9: 567" => singles in Stedman Triples.  Defaults to "1234".',
+              "0:1678" => 6ths place lead end single.  "-1: 3.123" => a Grandsire Single. \
+              "20:7890" => a 7890 single taking effect 20 changes into a lead (the Half Lead for \
```

Comment:
Is it worth just recommending a space in all examples?

Reply:
Hmmm I didn't really want to because it made the docs longer than they need to be.  But thinking about it more, the `--help` message is far too long _anyway_ and it feels better to make the docs consistent and fix the problem at its source.